examples/src/bin/builders.rs

```diff
@@ -0,0 +1,48 @@
+//! # Builders Sample
+//!
+//! This sample demonstrates how to create a widget using the builders.
+//! These allow to set construct-only properties and other construct
+//! properties when creating the widget.
+
+use gio::prelude::*;
+use gtk::prelude::*;
+
+use gtk::{Align, Application, ApplicationWindowBuilder, ButtonBuilder};
+
+use std::env::args;
+
+fn build_ui(application: &Application) {
+    let window = ApplicationWindowBuilder::new()
+        .application(application)
+        .title("First GTK+ Program")
+        .halign(Align::Center)
+        .valign(Align::Center)
+        .default_width(350)
+        .default_height(70)
+        .build();
+
+    let button = ButtonBuilder::new()
+        .margin_top(10)
+        .margin_bottom(10)
+        .margin_start(10)
+        .margin_end(10)
+        .visible(true)
```

Comment:
```suggestion
```
widgets are visible by default in gtk4

Reply:
You probably want to set the halign/valign to the button if you want to center it inside the window instead of it taking the whole width/height